hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/customrenderers/client/renderers/LinkRenderer.java

```diff
@@ -33,10 +46,19 @@ private void applystyle(Button button) {
         button.addStyleName(getStyle("borderless"));
         button.addStyleName(getStyle("small"));
         button.addStyleName(getStyle("on-focus-no-border"));
-        button.addStyleName(getStyle("link"));
     }
-    
+
     private String getStyle(final String style) {
         return new StringBuilder(style).append(" ").append(VButton.CLASSNAME).append("-").append(style).toString();
     }
+
+    private Map<String, String> formatData(String input) {
+        Map<String, String> details = new HashMap<>();
+        String[] tempData = input.split(",");
+        for (String targetData : tempData) {
+            String[] targetDataList = targetData.split(":");
```

Comment:
Is this defintely always an array of two elements? Because we hard accessing by index zero and one, what if in the value is no colon to split? Then there will no index one, maybe we should ensure it?